File: noi.openjudge.cn/ch0103/09.cpp
// http://noi.openjudge.cn/ch0103/solution/38830658/

#include<iostream>
using namespace std;
int main() {
    double r,d,s,c;
    double Pi = 3.14159;
    cin >> r;
    d = r * 2;
    s = Pi * r * r;
    c = Pi * d;
    printf("%0.4lf %0.4lf %0.4lf\n",d,c,s);
}

File: noi.openjudge.cn/ch0105/09.py
# http://noi.openjudge.cn/ch0105/solution/39346053/

# n, m = [int(s) for s in input().split()]
m, n = map(int, input().split())
b = 0
for x in range(m,n+1):
    if m % 2 == 1:
        b += m
    m += 1
print(b)
